alembic/versions/167243c0e86c_add_address_fields_for_groups.py

```diff
@@ -0,0 +1,73 @@
+"""Add address fields for groups
+
+Revision ID: 167243c0e86c
+Revises: b6074f8ea4ab
+Create Date: 2017-07-12 22:34:43.889923
+
+"""
+
+
+# revision identifiers, used by Alembic.
+revision = '167243c0e86c'
+down_revision = 'b6074f8ea4ab'
```

Comment:
Since we just merged https://github.com/magfest/ubersystem/pull/2673 in, I edited this revision to point to the migration that it adds to our system. Otherwise we'd have a branching upgrade path. Was that not correct?